zerver/lib/zulip_update_announcements.py

```diff
@@ -0,0 +1,207 @@
+    administrators = list(realm.get_human_admin_users())
+    with override_language(realm.default_language):
+        if realm.zulip_update_announcements_stream is None:
+            content = (
+                _(
+                    "You can now [configure]({organization_settings_url}) a stream where Zulip "
+                    "will send updates to users in your organization about new Zulip features."
+                )
+                + " "
+                + _("These notifications are currently turned off in your organization.")
+            ).format(organization_settings_url="/#organization/organization-settings")
+        else:
+            content = (
+                _(
+                    "Users in your organization will now receive updates about new features of Zulip "
+                    "in #**{zulip_update_announcements_stream}>{topic_name}**."
+                )
+                + "\n"
+                + "\n"
+                + _(
+                    "If you like, you can [configure]({organization_settings_url}) a different stream for "
+                    "these updates (and [move]({move_content_another_stream_help_url}) the initial updates there), "
+                    "or [turn this feature off]({organization_settings_url}) altogether."
+                )
+            ).format(
+                zulip_update_announcements_stream=realm.zulip_update_announcements_stream.name,
+                topic_name=Realm.ZULIP_UPDATE_ANNOUNCEMENTS_TOPIC_NAME,
+                organization_settings_url="/#organization/organization-settings",
+                move_content_another_stream_help_url="/help/move-content-to-another-stream",
+            )
+    return internal_prep_huddle_message(realm, sender, content, recipient_users=administrators)
+
+
+def is_group_direct_message_sent_to_admins_atleast_one_week_ago(realm: Realm) -> bool:
+    level_none_to_one_auditlog = RealmAuditLog.objects.filter(
+        realm=realm,
+        event_type=RealmAuditLog.REALM_PROPERTY_CHANGED,
+        extra_data__contains={
+            RealmAuditLog.OLD_VALUE: None,
+            RealmAuditLog.NEW_VALUE: 1,
+            "property": "zulip_update_announcements_level",
+        },
+    ).first()
+    assert level_none_to_one_auditlog is not None
+    group_direct_message_sent_on = level_none_to_one_auditlog.event_time
+    return timezone_now() - group_direct_message_sent_on > timedelta(days=7)
+
+
+def internal_prep_zulip_update_announcements_stream_messages(
+    current_level: int, latest_level: int, sender: UserProfile, realm: Realm
+) -> List[Optional[SendMessageRequest]]:
+    message_requests = []
+    stream = realm.zulip_update_announcements_stream
+    assert stream is not None
+    with override_language(realm.default_language):
+        topic_name = str(realm.ZULIP_UPDATE_ANNOUNCEMENTS_TOPIC_NAME)
+        while current_level < latest_level:
+            content = _(get_zulip_update_announcements_message_for_level(level=current_level + 1))
```

Comment:
What is this `_()` wrapping supposed to do? Internationalization doesn't work that way; see 
https://zulip.readthedocs.io/en/latest/translating/internationalization.html, but in short, `_()` needs to always wrap an actuals tring for it to be tagged for translation, because the discovery code actually parses the Python looking for specific patterns.

I worry a bit that it won't make sense to tag these updates for translation, because they are quite unlikely to be translated during the time between when we draft them and when they are published. So it's possible we should just not be translating the topic either...

Reply:
Chatted with @alya and she agrees probably we shouldn't attempt to translate the updates themselves, but the topic should be translated as currently implemented. 

So I think you can just delete the `_()` here and leave a comment in the `zulip_update_announcements` definition.

Reply:
> because they are quite unlikely to be translated during the time between when we draft them and when they are published.
> but the topic should be translated as currently implemented.

Are we sure about translating "topic names" before we make it available for cloud customers ?

Reply:
Made the changes to translate only the "topic name". The updates itself are not translated.